hctl

```diff
@@ -108,6 +111,7 @@ commands_usage() {
 while true ; do
     case $1 in
         -h|--help|help) usage; exit ;;
+        --systemd) systemd=true; ;;
```

Comment:
From the UX perspective it looks like a hope for a smart user (i.e. the user understands if there is systemd available or not). Usually it is not the best assumption 😁 

Is it possible to avoid explicit CLI argument and make the script smarter instead?

Reply:
Hmm, let me see.

Reply:
Update the patch.